python/spark_sklearn/test_utils.py

```diff
@@ -16,21 +16,32 @@
 from scipy.sparse import csr_matrix
 
 from pyspark import SparkContext
-from pyspark.sql import SQLContext
-from pyspark.mllib.linalg import Vectors
+from pyspark.sql import SparkSession
+from pyspark.ml.linalg import Vectors
 
-sc = None
+# Used as deocrator to have one PySpark SparkSession per fixture.
```

Comment:
Also, please mention in the doc above that it wraps `setUpClass` and `teadDownClass`, and that `spark` will contain the spark session.